.github/workflows/intergration-test.yml

```diff
@@ -0,0 +1,35 @@
+name: "Integration Test"
+
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Why we need `workflow_dispatch` here?

Reply:
This was added because I needed to trigger it manually, but currently I don't need it.